src/server/goodid/veramo.js

```diff
@@ -56,15 +58,21 @@ export const getAgent = once(async () => {
           ...keyDidResolver()
         })
       }),
-      new CredentialPlugin()
+      new CredentialPlugin(),
+      new CredentialIssuerLD({
```

Comment:
this wasn't documented. found it in the veramo packages' sources, inside tests folder